### PR TITLE
Add pages for education panels

### PR DESCRIPTION
**Pull request by meatballs**
https://github.com/PyconUK/2017-scheduler/issues/91

### CODE REVIEW

File: sessions/panel/the-ethics-and-politics-of-programming.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title: Panel Discussion - The Ethics and Politics of Programming
+subtitle: 
+speaker: nicholas-tollervey
```
Comment (ntoll) [inline sessions/panel/the-ethics-and-politics-of-programming.md]: Please update this to something like: `speaker: Carol Willing (chair)`

File: sessions/panel/python-in-education.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+title: Panel Discussion - Python in Education
+subtitle: 
+speaker: nicholas-tollervey
```
Comment (ntoll) [inline sessions/panel/python-in-education.md]: Please update this to something like: `speaker: Daniel Pope (chair)`

File: sessions/panel/developer-career-progression.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+title: Panel Discussion - Developer Career Progression
+subtitle: 
+speaker: nicholas-tollervey
```
Comment (ntoll) [inline sessions/panel/developer-career-progression.md]: Please update this to something like: `speaker: Samathy Barratt (chair)`

**Comment by ntoll** [comment]
Thank you! Would it be possible to add some notion that the persons designated as speakers are, in fact, acting in the capacity of chair? Not sure how that would happen, but perhaps in the subtitle..? "Chaired by..." Thoughts?

**Comment by meatballs** [comment]
I've done the basics here - whatever is needed for the content is somebody else's problem

**Comment by ntoll** [comment]
Aha. Understood.

**Pull request merged**